pkg/addon/deleteaddon/deleate_addon.go

```diff
@@ -0,0 +1,36 @@
+package deleteaddon
```

Comment:
Could you please delete this file now that the uninstall is happening in the multiclusterhub-operator?